federated_content_connector/filters/pipeline.py

```diff
@@ -36,4 +35,4 @@ def run_filter(self, course_key, course_home_url):
             if course_type == EXEC_ED_COURSE_TYPE and product_source == PRODUCT_SOURCE_2U:
                 return {'course_key': course_key, 'course_home_url': EXEC_ED_LANDING_PAGE}
```

Comment:
I think we should change the `course_home_url` value here only. There should be only one return statement.